### ISSUE TITLE
Key Concepts and Data Model

### ISSUE THREAD

**Issue by rufuspollock** [rgrp/climate-negotiations#4]
Data Model
- Actor
  - countries (e.g. France, Saudi Arabia, China - we have a dictionary for this)
  - negotiating groups (e.g. G77 and China, AOSIS, - we have a dictionary for this)
- Document - the ENB published documents
  - Date, Title
  - => event (each document is only ever associated with one event)
  - URL on the ENB website
- Event
  - Date (generally the month and the year are enough to distinguish an event)
  - Place (the city where the meeting took place)
- Sections - essentially correspond to the meeting of a particular track during an event (e.g. the meeting of SBI in Bonn in june 2014) or to a particular reporting bit on an event (e.g. the analysis of one day or the feelings in the corridors)
  - Tracks (generally 1, but there may joint meetings such as SBSTA + SBI )
  - Reporting Format (1)
  - Document - note we may inline certain document and document->event properties e.g.
    - Date
    - Place (of meeting)
- Text chunks - essentially correspond to a particular discussion during a session at an event
  - Section (1)
  - Actors (0 ≤ actors ≥ as many as mentioned)
  - Topics (0 ≤ actors ≥ 3 most prominent)

Classifications:
- Type - aka Negotiation Track
- Subtype - aka Reporting Formats
- Topics (e.g. Forest, Mitigation, Adaptation)
  For reference existing CSV "database" on the climatenegotiations.org site has:

```
id,title,actors,countries,topics,url,event_id,track,format,date,year,fulltext
```
## Enumerations

Type (negotiation tracks)
- COP  
- IPCC  
- SBI  
- SBI/SBSTA (these are joint meeting, it may be more intelligent to tag them with both SBI and SBSTA)
- SBSTA  
- Special workshop
- Working groups
  - BUT is may be better to separate the different working groups
- undeducible
- blank

Subtypes (reporting formats)
- Agenda
- Analysis 
- Corridors 
- History 
- Groups (contact groups, informal groups, non-groups...)
- Plenary
- undeducible
- blank

**Comment by tommv** [comment]
* Negotiation tracks (type)
* Reporting formats (subtype)

* Topics
* Actors 
   * Countries
   * Negotiating groups

* Date
* Place

* URL

**Comment by rufuspollock** [comment]
@tommv i've refined this a bit. Does it make sense - e.g. the Meeting / Event concept. What attributes do we have re meetings / events (cf #1)

**Comment by tommv** [comment]
Yes it makes perfect sense. I corrected a few details, but not much.
Most important edits I made are:
- Removed "Participants" there may be way to get this but not in the data we have so far
- Distinguished "Sections" (larger) from "Text chunck" (smaller) (cfr. #13 )
- Added the possibility of multiple tag of "tracks" for "sections" (to handled joint meeting)
